Nos/Views/Onboarding/CreateAccountView.swift

```diff
@@ -0,0 +1,105 @@
+import Dependencies
+import SwiftUI
+
+/// The Create Account view in the onboarding.
+struct CreateAccountView: View {
+    @Environment(OnboardingState.self) private var state
+
+    @Dependency(\.crashReporting) private var crashReporting
+    @Dependency(\.currentUser) private var currentUser
+
+    var body: some View {
+        GeometryReader { geometry in
+            ScrollView {
+                VStack(alignment: .leading, spacing: 20) {
```

Comment:
Can we put .readabilityPadding() on this for the Mac/iPad?

Reply:
Yes!